src/main/cli/cli.c

```diff
@@ -1500,10 +1500,10 @@ static void cliSerialPassthrough(char *cmdline)
 
     char *resetMessage = "";
     if (port1ResetOnDtr && ports[1].id == SERIAL_PORT_USB_VCP) {
-        resetMessage = "or drop DTR ";
+        resetMessage = ", or drop DTR ";
     }
 
-    cliPrintLinef("Forwarding, power cycle %sto exit.", resetMessage);
+    cliPrintLinef("Forwarding. Power cycle, or use +++ATH %sto exit.", resetMessage);
```

Comment:
(missing space in %sto)

Reply:
The extra space is in `resetMessage` a couple of lines up.